src/main/java/org/broadinstitute/hellbender/utils/locusiterator/LocusIteratorByState.java

```diff
@@ -342,7 +342,8 @@ private void lazyLoadNextAlignmentContext() {
                 }
 
                 if (!pile.isEmpty()) { // if this pileup added at least one base, add it to the full pileup
-                    fullPileupPerSample.put(sample, new ReadPileup(location, pile));
+                    // the pileup for LIBS is sorted by coordinate, because it comes from a sorted BAM file
```

Comment:
This was changed by the performance improvement for HC in #4031 - we should check if maintaining the stratified pileup here cause a regression of that improvement.